src/cooperative.jl

```diff
@@ -38,11 +38,12 @@ function build_specific_model!(::AbstractGroupCO, ECModel::AbstractEC)
     market_data = ECModel.market_data
 
     n_users = length(users_data)
+    u_standard = first(keys(users_data))
```

Comment:
As specified above, avoid using this u_standard as the first one

Reply:
Removed

Reply:
Maybe you needa push^?